src/HowItWorks/HowItWorksScreen.tsx

```diff
@@ -272,6 +273,6 @@ const dotsStyle = StyleSheet.create({
   },
 })
 
-const MemoizedOnboardingScreen = React.memo(OnboardingScreen)
+const MemoizedHowItWorksScreen = React.memo(HowItWorksScreen)
```

Comment:
Could it be that the problem we were having with having to add 2 stacks for the onboarding had to do with not receiving the navigation as a prop here? Am I way off? Curious if accessing the navigation via the hook and doing the render based on the props had something to do with it. If you tried it sounds like a nice learning. 😄

Reply:
That's an interesting thought, but I don't think so. The problem with having two stacks with the same name was a bit different. The onboarding stack that is shown when the app is first downloaded was getting registered as the initial route, and since another stack with that same name was in the main app stack, the name did not get unregistered as the initial route -- even though it wasn't the first screen in the navigator. Does that make sense?